services/121-service/src/utils/formatDate.ts

```diff
@@ -0,0 +1,11 @@
+function padTo2Digits(num: number): string {
+  return num.toString().padStart(2, '0');
+}
+
+export function formatDate(date: Date): string {
```

Comment:
Refactor: we have various "formatDate" functions throughout our code base, and AFAIU there is also a function in the framework with the same name we use. To reduce complexity/confusion: refactor. If this is a very specific function to format a date for Safaricom only, I would say: put it in the Safaricom Module and make the name more specific.